pkg/oras/download.go

```diff
@@ -0,0 +1,70 @@
+package oras
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/aws/eks-anywhere/pkg/curatedpackages"
```

Comment:
Is there a way we can avoid this dependency?
`oras` is a "generic" package (that can be used for storage operations in oci registries)

If this was intended to be curated packages exclusive, can we move it to `pkg/curatedpackages/oras`? Or even `pkg/curatedpackages/bundles` or something like that (since it's no actually doing anything oras specific